model-transformations/PropertyCompositionToAssociation.md

```diff
@@ -19,6 +19,52 @@
 <tr>
 <td>Original instance in default encoding:</td>
 <td>
+<p>Possibility 1: inline</p>
+
+```xml
+<am:ManagementRestrictionOrRegulationZone>
+  <!-- ... -->
+  <am:legalBasis>
+    <base2:LegislationCitation>
+      <base2:name>Bekendtgørelse af lov om skove</base2:name>
+      <base2:shortName>LBK nr 122 af 26/01/2017</base2:shortName>
+      <base2:date>
+        <gmd:CI_Date>
+          <gmd:date>
+            <gco:Date>2017-01-26</gco:Date>
+          </gmd:date>
+          <gmd:dateType>
+            <gmd:CI_DateTypeCode
+              codeListValue="creation"
+              codeList="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/codelist/ML_gmxCodelists.xml#CI_DateTypeCode" />
+          </gmd:dateType>
+        </gmd:CI_Date>
+      </base2:date>
+      <base2:link>http://www.retsinformation.dk/eli/lta/2017/122</base2:link>
+      <base2:level
+        xlink:href="http://inspire.ec.europa.eu/codelist/LegislationLevelValue/national"
+        xlink:title="national" />
+    </base2:LegislationCitation>
+  </am:legalBasis>
+  <!-- ... -->
+</am:ManagementRestrictionOrRegulationZone>
+```
+
+<p>Possibility 2: by reference</p>
+
+```xml
+<am:ManagementRestrictionOrRegulationZone>
+  <!-- ... -->
+  <am:legalBasis
+    xlink:href="http://www.retsinformation.dk/eli/lta/2017/122"
+    xlink:title="Bekendtgørelse af lov om skove" />
+  <!-- ... -->
+</am:ManagementRestrictionOrRegulationZone>
+```
```

Comment:
Would this indeed be an allowable encoding? My understanding is that in the existing encoding, we would need to have a reference that points to a GML object of the type `LegislationEncoding`.

I did see now that the example that is currently in the master is basically what you list as "possiblity 3", but given that `http://www.retsinformation.dk/eli/lta/2017/122` does not seem to reference a valid GML object, I am wondering about the validity of this.